docs/spark/spark-writes.md

```diff
@@ -311,7 +311,9 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
+To write with SQL, both global sort (`ORDER BY`) and local sort (`SORT BY`) work for the requirement.
```

Comment:
"the requirement" was always a bit confusing to me here. What do you think about this re-wording?

> With SQL, you can use `ORDER BY` to achieve global sorting or `SORT BY` to achieve local sorting.
> 
> With dataframes, you can use `orderBy`/`sort` to achieve global sorting or `sortWithinPartitions` to achieve local sorting.